specs/trs/tracked-resource-set.html

```diff
@@ -423,15 +423,28 @@ <h2>Base Segmentation</h2>
       <p>
         The number of Tracked Resources in the Base could be such that it is not reasonable to list them all in a single
         HTTP response. In this case,
-        <span class="conformance"
-          >the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
-          listing a subset of all Tracked Resources</span
-        >.
+        <span class="conformance">
+          the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
```

Comment:
The segmentation is not specified here properly. Can there be triple overlap or are triples _partitioned_ into segments (pages)? Can a single RDF resource have triples split across pages?

Reply:
If you mean, can statements against a specific subject resource be split across multiple pages. Yes. And that's always been the case with both OSLC paging and TRS base paging. The proposal does not make any semantic changes here.